plugins/toolbox/src/components/Root/ToolsPage.tsx

```diff
@@ -326,7 +334,12 @@ export const ToolsPage = (props: ToolsPageProps) => {
                       >
                         {tool.headerButtons}
                         {tool.showOpenInNewWindowButton !== false && (
-                          <Tooltip title="Open tool in new window" arrow>
+                          <Tooltip
+                            title={i18n_UNSAFE(
```

Comment:
Is unsafe necessary?

Reply:
Leave for now until backstage fixes the types.